Admin.js

```diff
@@ -331,7 +331,7 @@ var Admin = {
             Admin.stopEvent(event);
 
             var container = jQuery(this).closest('[data-prototype]');
-            var counter = ++Admin.collectionCounters[container.attr('id')];
+            var counter = ++Admin.lolCounters[container.attr('id')];
```

Comment:
- eslint: Unexpected var, use let or const instead.
- eslint: Combine this with the previous 'var' statement.
- eslint: All 'var' declarations must be at the top of the function scope.
- eslint: Unary operator '++' used.
- eslint: Strings must use doublequote.